trojsten/submit/models.py

```diff
@@ -107,9 +107,10 @@ class Submit(models.Model):
         )
     )
     protocol_id = models.CharField(
-        max_length=128, verbose_name='číslo protokolu', blank=True)
+        db_index=True, max_length=128, verbose_name='číslo protokolu', blank=True)
 
     reviewer_comment = models.TextField(verbose_name='komentár od opravovateľa', blank=True)
+    protocol = models.TextField(verbose_name='protokol', blank=True, null=True)
```

Comment:
chceme drzat protokoly v DB? momentalne nam protokoly zaberaju na disku okolo 500MB

Reply:
Otazka je co s nimi - akoze jeden protocol ma zvycajne <1kb, co nie je az tak vela ze by stalo za to to komplikovat filesystemom.
Ak ich nutne nepotrebujeme, mozme protocol sparsovat a do db ulozit len relevantne info.
To by malo usetrit znacnu cast miesta.

Reply:
no moj povodny napad bol ukladat ich do django storage (co pre development moze byt filesystem a pre produkciu napr. google cloud bucket)

ale postgres zvladne aj niekolko GB databazy v pohode bez straty vykonu ak to nie je index, takze za mna to moze ostat aj v DB.., s tym, ze napr. ak by nam vadil narast velkosti databaze tak mozme odmazat 5+ rocne protokoly napr.